docs/dashboard/home.md

```diff
@@ -60,4 +60,14 @@ If you prefer to work in dark mode, click on your account name in the top right
 
 ### How to log out
 
-To log out, click your account name in the top right corner of the dashboard, and then click the _log out_ button. 
+To log out, click your account name in the top right corner of the dashboard, and then click the _log out_ button.
```

Comment:
<img width="686" alt="image" src="https://user-images.githubusercontent.com/5188636/120191877-c7165580-c21a-11eb-9592-7707043844f5.png">

Available actions should be described here

Reply:
Comment should be to navigator section description.